test/e2e/storage/vsphere/vsphere_stress.go

```diff
@@ -114,21 +114,21 @@ var _ = utils.SIGDescribe("vsphere cloud provider stress [Feature:vsphere]", fun
 		var wg sync.WaitGroup
 		wg.Add(instances)
 		for instanceCount := 0; instanceCount < instances; instanceCount++ {
-			instanceId := fmt.Sprintf("Thread:%v", instanceCount+1)
-			go PerformVolumeLifeCycleInParallel(f, client, namespace, instanceId, scArrays[instanceCount%len(scArrays)], iterations, &wg)
+			instanceID := fmt.Sprintf("Thread:%v", instanceCount+1)
+			go PerformVolumeLifeCycleInParallel(f, client, namespace, instanceID, scArrays[instanceCount%len(scArrays)], iterations, &wg)
 		}
 		wg.Wait()
 	})
 
 })
 
-// goroutine to perform volume lifecycle operations in parallel
-func PerformVolumeLifeCycleInParallel(f *framework.Framework, client clientset.Interface, namespace string, instanceId string, sc *storagev1.StorageClass, iterations int, wg *sync.WaitGroup) {
+// PerformVolumeLifeCycleInParallel goroutines to perform volume lifecycle operations in parallel
```

Comment:
nit: this function is only in a goroutine if a caller adds `go` in front. In actuality, this function doesn't do anything in parallel. It would be more accurate to say that this function performs a complete volume lifecycle N times, serially. (And I would love it if we could rename the function.)

Reply:
OK. Should I add the modified lines into a new commit or amend into the commit before ?

Reply:
fixed

Reply:
I prefer git amend rather than adding commits, because at the end of the day, we have a git log and not a github pull request log.